antora-ui-camel/src/partials/header-content.hbs

```diff
@@ -15,6 +15,10 @@
                 {{/each}}
                 </div>
                 </div>
+                <div class="navbar-search results-hidden">
+                  <input id="search_mobile" class="search" placeholder="Search" autocomplete="off">
+                  <div id="search_results_mobile"></div>
+                </div>
```

Comment:
I think problem with this is that it's rendered multiple times (notice the `#each @items` on line 8.